docs/source/include/learning.rst

```diff
@@ -0,0 +1,93 @@
+Learned Models
+==============
+
+In SCICO, neural network models are used to represent imaging problems and provide different modes of data-driven regularization.
+The models are implemented in `FLAX <https://flax.readthedocs.io/>`_, and constitute a representative sample of frequently used networks.
```

Comment:
It looks as if it's usually written `Flax` rather than `FLAX`.

Reply:
Changed the entry to Flax.